src/components/Maps/Sidebar.tsx

```diff
@@ -37,30 +63,67 @@ const useStyles = makeStyles({
   },
 });
 
+const formatDateToString = (date: Date | null) => {
+  if (!date) {
+    return '';
+  }
+  const dateParts: {
+    year?: string;
+    month?: string;
+    day?: string;
+  } = new Intl.DateTimeFormat().formatToParts(date).reduce(
+    (obj, currentPart) => ({
+      ...obj,
+      [currentPart.type]: currentPart.value,
+    }),
+    {}
+  );
+
+  if (dateParts.day) {
+    if (dateParts.day.length < 2) {
+      dateParts.day = '0'.concat(dateParts.day);
+    }
+  }
+
+  if (dateParts && dateParts.year && dateParts.month && dateParts.day) {
+    return dateParts.year.concat('-', dateParts.month, '-', dateParts.day);
+  }
+
+  return '';
+};
```

Comment:
can this be moved into https://github.com/uwblueprint/paramedics-react/blob/master/src/utils/format.ts